frontend/src/pages/admin/questions/create.tsx

```diff
@@ -62,10 +79,23 @@ const QuestionCreate: NextPage = (): JSX.Element => {
     /**
     * This will Save all the data seelcted
    */
-  const handleSave = (data: QuestionFormValues) => {
-    console.log('Category ID', data?.category_id)
-    console.log('Question: ', data?.value)
-    console.log('Choices: ', choices)
+  const handleSave = async (data: QuestionFormValues): Promise<void> => {
+    await 
+      axios
+        .post('/api/questions', {
+          category_id: data?.category_id,
+          value: data?.value,
+          choice_id: data?.choice_id,
+          choices
+        })
```

Comment:
you can destructure and then use ES6 coding convention
```suggestion
  const { category_id, value, choice_id } = data
    await 
      axios
        .post('/api/questions', {
          category_id,
          value,
          choice_id,
          choices
        })
```